### ISSUE TITLE
Move help button location to be after question title

### ISSUE THREAD

**Issue by shelaghm** [google/android-fhir#1965]
**Problem**
Current help button is located after instructions field, which looks awkward if there's no instructions text

**Describe the solution you'd like**
EmCare implementation moved the help button to be located after the question title. 

**Describe alternatives you've considered**
Could make this configurable, but don't think it is necessary at this time.

**Additional context**
EmCare commit https://github.com/google/android-fhir/commit/c130243616a9c355e8e3b5a9f25673e817769336

**Screenshot**
<img width="408" alt="Screenshot 2023-04-12 at 14 47 25" src="https://user-images.githubusercontent.com/86378921/231593295-ee4d8efd-dced-423a-a8bd-bbbc31d4b339.png">

cc @santosh-pingle

**Comment by santosh-pingle** [comment]
@kunjan8794 @khyativyasargus
I see corresponding commit of your repository https://github.com/google/android-fhir/commit/c130243616a9c355e8e3b5a9f25673e817769336
Are you going to create PR corresponding to main repository android fhir sdk?
Or you want me to look into it?
cc @jingtang10

**Comment by khyativyasargus** [comment]
@santosh-pingle Sure. We will create PR for this in main repository.

**Comment by santosh-pingle** [comment]
@khyativyasargus thanks!, is there any approximate date by which we can expect the pull request?

**Comment by khyativyasargus** [comment]
@santosh-pingle Yes, You can expect all PRs done by 24th April, Monday. Let me know if it's needed before that. Thanks!
